vhdbuilder/prefetch/internal/containerimage/testdata/prefetch.sh

```diff
@@ -19,12 +19,12 @@ prefetch() {
 }
 prefetch "mcr.microsoft.com/containernetworking/azure-cni:v1.5.35" "/dropgz"
 prefetch "mcr.microsoft.com/containernetworking/azure-cni:v1.5.32" "/dropgz"
-prefetch "mcr.microsoft.com/containernetworking/azure-cni:v1.6.10" "/dropgz"
+prefetch "mcr.microsoft.com/containernetworking/azure-cni:v1.6.11" "/dropgz"
 prefetch "mcr.microsoft.com/containernetworking/azure-cni:v1.6.7" "/dropgz"
 prefetch "mcr.microsoft.com/containernetworking/azure-cns:v1.4.52" "/usr/local/bin/azure-cns"
 prefetch "mcr.microsoft.com/containernetworking/azure-cns:v1.5.35" "/usr/local/bin/azure-cns"
 prefetch "mcr.microsoft.com/containernetworking/azure-cns:v1.5.32" "/usr/local/bin/azure-cns"
-prefetch "mcr.microsoft.com/containernetworking/azure-cns:v1.6.10" "/usr/local/bin/azure-cns"
+prefetch "mcr.microsoft.com/containernetworking/azure-cns:v1.6.11" "/usr/local/bin/azure-cns"
```

Comment:
did you update this manually?

Reply:
yes

Reply:
please run `make validate-prefetch` from the repo root , or `make generate` from within `vhdbuilder/prefetch` to make sure the content is properly in-sync: https://github.com/Azure/AgentBaker/blob/a3baa6bbf077d8ad57b2759b49c8c79360fedb94/parts/linux/cloud-init/artifacts/README-COMPONENTS.md?plain=1#L73C1-L74C1